packages/scene-composer/src/components/panels/scene-components/common/ValueDataBindingBuilder.scss

```diff
@@ -0,0 +1,8 @@
+.copy-select-flex {
+    display: flex;
+    flex-wrap: nowrap;
+}
+
+.full-width {
```

Comment:
prefix class name with `tm-` to avoid conflict with others

Reply:
He does not need this. He should be able to place components inline with space between.